share/picongpu/tests/CollisionsThermalisation/include/picongpu/param/particle.param

```diff
@@ -115,18 +133,29 @@ namespace picongpu
             template<uint32_t numParticles>
             struct OnePositionParam
             {
-                /** Count of particles per cell at initial state
+                /** Count of macro-particles per cell at initial state
                  *
                  *  unit: none */
                 static constexpr uint32_t numParticlesPerCell = numParticles;
 
                 const InCellOffset_t inCellOffset;
             };
-            // definition of quiet particle start
-            using Center1000ppc = OnePositionImpl<OnePositionParam<1000u>>;
-            using Center5000ppc = OnePositionImpl<OnePositionParam<5000u>>;
+            /** Definition of start position functor that
+             * places macro-particles at the initial in-cell position defined above.
+             * Number of macro-particles per cell is defined by argument to OnePositionParam<>.
+             */
+            using Corner1000ppc = OnePositionImpl<OnePositionParam<1000u>>;
+            using Corner5000ppc = OnePositionImpl<OnePositionParam<5000u>>;
 
         } // namespace startPosition
 
+        /** Number of maximum macro-particles per cell during density profile evaluation.
+         *
+         * Determines the weighting of a macro particle as well as the number of
+         * macro-particles which sample the evolution of the particle distribution
+         * function in phase space.
+         */
+        constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 1000u;
```

Comment:
same here

Reply:
this line moved from line 54, but is also seems to have changed, original value was 350 now it is 1000, @steindev was there any particular reason for that?

Reply:
Same as above. I probably changed it in order to match it to at least one of the possible values.